tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>Hmmm, no other tutorials have an Index like this. I'd like to discuss this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>We discussed this and decided to handle the Index as part of the template so every tutorial will have it automatically. </p>

Reply:
we also highlighted this as a nice low-hanging fruit issue to tackle in e.g. https://github.com/astropy/learn-astropy/issues/26